packages/languages/src/browser/semantic-highlighting/semantic-highlighting-feature.ts

```diff
@@ -23,7 +23,7 @@ import {
     ClientCapabilities,
     ServerCapabilities,
     Disposable,
-    DocumentSelector
+    DocumentSelector,
```

Comment:
[minor] the change is unnecessary

Reply:
Left over from typing tests, will remove.